tools/cldr-apps/src/main/java/org/unicode/cldr/web/SubtypeToURLMap.java

```diff
@@ -199,11 +199,18 @@ public List<String> getList() {
      * @throws IllegalArgumentException
      */
     protected SubtypeToURLMap(BufferedReader utf8Data) throws IllegalArgumentException{
-        URLMapReader mapReader = new URLMapReader();
-        mapReader.read(utf8Data);
-
-        this.map = mapReader.getMap();
-        this.urlList = mapReader.getList();
+        try {
+            URLMapReader mapReader = new URLMapReader();
+            mapReader.read(utf8Data);
+
+            this.map = mapReader.getMap();
+            this.urlList = mapReader.getList();
+        } catch(Throwable t) {
+            t.printStackTrace();
+            System.err.println("Error " + t +" constructing SubtypeToURLMap, using empty SubtypeToURLMap");
+            this.map = new HashMap<>();
+            final List<String> newList = new ArrayList<>();
```

Comment:
newList seems superfluous?

Reply:
good catch. that was supposed to be `this.urlList = new ArrayList…` … fixed